app/test/frontend/golden/v2/search_page.html

```diff
@@ -91,6 +91,7 @@ <h1 class="_visuallyhidden">Dart pub</h1>
     <main class="package-banner">
       <form class="search-bar" action="/experimental/search">
         <input class="input" name="q" placeholder="Search packages" autocomplete="on" autofocus value="foobar">
+        
```

Comment:
shouldn't the hidden input show up here? Or just not yet?

Reply:
It is a conditional part of the template, because there is no reason to put the `platform` parameter in the URL if there is no platform set.

Reply:
Ah...k

Reply:
So we just don't have a `golden` that causes the input to be rendered?

Reply:
yeah, I'll add the golden files with the landing blurb change (mostly because I wanted to spare a few merge conflicts)